src/assignment/assignment.service.ts

```diff
@@ -1048,4 +1126,151 @@ export class AssignmentService {
 			});
 		}
 	}
+
+	static async fetchPupilCollectionsForAdmin(
```

Comment:
Lijkt me best om dit in een aparte service `pupil-collections.service` & module te zetten